homeassistant/components/aladdin_connect/config_flow.py

```diff
@@ -0,0 +1,90 @@
+"""Config flow for Whirlpool Laundry integration."""
```

Comment:
Doesn't seem this is Whirlpool

Reply:
Maytag and Whirlpool are the same company.